[docs] Update note on vulnerabilities (#64822)
ray-project/ray: doc/source/ray-security/index.md

```

::::{note}

> Find the latest Ray security alerts at [Security Advisories](https://github.com/ray-project/ray/security/advisories) on GitHub. Use the [**Report a vulnerability**](https://github.com/ray-project/ray/security/advisories/new) form or email security@anyscale.com to report a new security issue.

Reviewer: I would do: 

- Use the [**Report a vulnerability**](https://github.com/ray-project/ray/security/advisories/new) form or email security@anyscale.com to report a new security issue.
- Find the latest Ray security alerts at [Security Advisories](https://github.com/ray-project/ray/security/advisories) on GitHub.

> Find the latest Ray security alerts at [Security Advisories](https://github.com/ray-project/ray/security/advisories) on GitHub. Use the [**Report a vulnerability**](https://github.com/ray-project/ray/security/advisories/new) form or email security@anyscale.com to report a new security issue.

Author: ```suggestion
Use the [**Report a vulnerability**](https://github.com/ray-project/ray/security/advisories/new) form or email [security@anyscale.com](mailto:security@anyscale.com) to report a new security issue. For the current Ray security alerts, check [Security Advisories](https://github.com/ray-project/ray/security/advisories) on GitHub. 
```

> Use the [**Report a vulnerability**](https://github.com/ray-project/ray/security/advisories/new) form or email [security@anyscale.com](mailto:security@anyscale.com) to report a new security issue. For the current Ray security alerts, check [Security Advisories](https://github.com/ray-project/ray/security/advisories) on GitHub. 

Author: ```suggestion
Use the [**Report a vulnerability**](https://github.com/ray-project/ray/security/advisories/new) form or email [security@anyscale.com](mailto:security@anyscale.com) to report a new security issue. For the current Ray security alerts, check [Security Advisories](https://github.com/ray-project/ray/security/advisories) on GitHub.
```
